src/iuse.cpp

```diff
@@ -4106,8 +4106,10 @@ int iuse::mp3( player *p, item *it, bool, const tripoint & )
         p->add_msg_if_player( m_info, _( "You put in the earbuds and start listening to music." ) );
         if( it->typeId() == "mp3" ) {
             it->convert( "mp3_on" ).active = true;
+            p->moves -= to_moves<int>( 1_seconds );
```

Comment:
Please use `Creature::mod_moves` instead.

Reply:
OK, would this be equivalent? `p->mod_moves( -100 )`

Reply:
Yes, but you can still write it like this: `p->mod_moves( -to_moves<int>( 1_seconds ) )`. I guess we should make an overloaded version of `mod_moves` which accepts `time_duration` directly, but doesn't allow granting moves. Maybe even rename it to something like `consume_moves( 1_second )`.